mivs/site_sections/mivs_applications.py

```diff
@@ -73,12 +73,16 @@ def game(self, session, message='', **params):
         }
 
     def developer(self, session, message='', **params):
-        developer = session.indie_developer(params, applicant=True)
+        developer = session.indie_developer(params, applicant=True, bools=["primary_contact"])
         if cherrypy.request.method == 'POST':
             message = check(developer)
             if not message:
-                session.add(developer)
-                raise HTTPRedirect('index?message={}', 'Presenter added')
+                primaries = session.query(IndieDeveloper).filter_by(studio_id=developer.studio_id, primary_contact='true').all()
```

Comment:
Although ``primary_contact='true'`` works, the more correct form of this is ``primary_contact=True``.  Can you make that change?  Other than that this looks good to go!